pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -766,7 +766,7 @@
     "sensor.cups" = ps: with ps; [ pycups ];
     "sensor.currencylayer" = ps: with ps; [  ];
     "sensor.daikin" = ps: with ps; [  ];
-    "sensor.darksky" = ps: with ps; [  ];
+    "sensor.darksky" = ps: with ps; [ python-forecastio ];
```

Comment:
Was this generated by `parse-requirements.py`? The top of the file says "do not edit".

Reply:
oh oh :D i will look into it

Reply:
I ran the command, however it was unable to detect the new dependency. now sure what to touch to make it work.
The package is called python-forecastio and will be installed with `pip install python-forecastio`. (see https://github.com/ZeevG/python-forecast.io )
can you check the parse-requirements.py package what went wrong there?

Reply:
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/darksky.py#L22 requires it

Reply:
exactly, but how can i get the `parse-requirements.py` to find this requirement and add it automatically to the list of components?

Reply:
cc @peterhoeg @dotlambda